utils/config.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
   secret,
   secretLogs,
   sessionDurationHours: process.env.SESSION_DURATION_HOURS || '2', // duration in hours
+  refreshDurationHours: process.env.REFRESH_DURATION_HOURS || 1, // duration in hours
```

Comment:
We may add this env var in .env.sample to keep it as the lsit of all possible env vars.